pages/examples/content/es/css-grid.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  title: 'CSS Grid',
+  subtitle: 'La mejor solución para el diseño CSS',
+  description: `
+    [CSS Grid Layout](https://developer.mozilla.org/fr/docs/Web/CSS/CSS_Grid_Layout)
```

Comment:
Since these docs are in Spanish I think it would be better to refer the Spanish mozilla docs @ [https://developer.mozilla.org/es/docs/Web/CSS/CSS_Grid_Layout](https://developer.mozilla.org/es/docs/Web/CSS/CSS_Grid_Layout)